src/plots/cartesian/select.js

```diff
@@ -0,0 +1,102 @@
+/**
+* Copyright 2012-2015, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+var polygon = require('../../lib/polygon');
+var axes = require('./axes');
+var filteredPolygon = polygon.filter;
+var polygonTester = polygon.tester;
+var BENDPX = 1.5; // max pixels off straight before a line counts as bent
+
+function getAxId(ax) { return ax._id; }
+
+module.exports = function prepSelect(e, startX, startY, dragOptions, mode) {
```

Comment:
Nice.

How hard would it be the split up the [`dragBox`](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/graph_interact.js#L1309) definition out of `graph_interact.js` and into a separate file?